app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class SessionsController < ApplicationController
+  before_action :load_user, only: :create
+
+  def new; end
+
+  def create
+    if @user&.authenticate(params.dig(:session, :password))
```

Comment:
```suggestion
    if @user.authenticate params.dig(:session, :password)
```
Xóa dc vì ở đây chắc chắn user khác nil rồi